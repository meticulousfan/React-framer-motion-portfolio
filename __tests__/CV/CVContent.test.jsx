/**
 * @jest-environment jsdom
 */

import CVContent from "../../src/components/CV/CVContent.component";
import { render } from "@testing-library/react";

describe("CVContent", () => {
  it("CVContent laster inn og kan vises", () => {
    const { container } = render(<CVContent />);
    expect(container).toMatchSnapshot();
  });
});
