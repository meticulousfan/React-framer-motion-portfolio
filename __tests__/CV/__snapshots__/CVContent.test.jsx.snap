// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CVContent CVContent laster inn og kan vises 1`] = `
<div>
  <main
    id="maincontent"
  >
    <div
      class="mt-32 bg-graybg"
    >
      <div
        class="px-4 p lg:px-0 xl:px-0 md:px-0"
      >
        <div
          class="container mx-auto bg-white rounded shadow"
        >
          <div
            class="p-4 mx-auto mt-4"
          >
            <div
              class="p-4 text-lg rounded"
            >
              <div
                class="mt-0 sm:mt-2 xs:mt-2"
              >
                <div
                  class="react-pdf__Document hidden mt-4 text-center xl:block lg:block"
                >
                  <div
                    class="react-pdf__message react-pdf__message--loading"
                  >
                    Laster inn PDF ...
                  </div>
                </div>
              </div>
              <div
                class="mx-auto mt-0 text-center sm:mt-2 xs:mt-2"
              >
                <button
                  class="p-4 m-4 text-white transition duration-500 ease-in-out bg-gray-800 rounded hover:shadow-outline hover:bg-gray-600"
                  type="submit"
                >
                  <a
                    href="/CV-dfweb.pdf"
                  >
                    Last ned PDF
                  </a>
                </button>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </main>
</div>
`;
